snapshot/lib/snapshot/detect_values.rb

```diff
@@ -23,10 +23,25 @@ def self.set_additional_default_values
         config[:devices] = []
 
         # We only care about a subset of the simulators
-        FastlaneCore::Simulator.all.each do |sim|
-          next if sim.name.include?("iPad") and !sim.name.include?("Retina") # we only need one iPad
-          next if sim.name.include?("6s") # same screen resolution
-          next if sim.name.include?("5s") # same screen resolution
+        all_simulators = FastlaneCore::Simulator.all
+        all_simulators.each do |sim|
+          # Filter iPads, we only want the following simulators
+          # Xcode 7:
+          #   ["iPad Pro", "iPad Air"]
+          # Xcode 8:
+          #   ["iPad Pro (9.7 Inch)", "iPad Pro (12.9 Inch)"]
+          #
+          # Full list: ["iPad 2", "iPad Retina", "iPad Air", "iPad Air 2", "iPad Pro"]
+          next if sim.name.include?("iPad 2")
+          next if sim.name.include?("iPad Retina")
+          next if sim.name.include?("iPad Air 2")
+          # In Xcode 8, we only need iPad Pro 9.7 inch, not the iPad Air
+          next if all_simulators.any? { |a| a.name.include?("9.7 inch") } && sim.name.include?("iPad Air")
```

Comment:
Yeah we could, however we're already pre-fetching the simulators into `all_simulators`, so that the bottle neck really is just the Ruby method for finding an element in an array 👍 
Do you think an additional optimisation is necessary?